### PR TITLE
website: updating troubleshooting guide

### LINKED ISSUE #750: add more content to troubleshooting guide
### Is your enhancement related to a problem? Please describe

Guide to identified common problem should be made available to the website

### Describe the solution you'd like

publish identified solutions to the website

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by deekay2310**
Signed-off-by: Dev Kumar <devskumar99@gmail.com>

### What does this PR do?
Added a section pertaining to Podman Engine in the troubleshooting guide

### Screenshot/screencast of this PR

Screen Shot 2022-11-03 at 11 08 59 AM

Screen Shot 2022-11-03 at 11 09 09 AM

Screen Shot 2022-11-03 at 11 09 21 AM

### CODE REVIEW

File: website/docs/troubleshooting.md [hunk 1]
```diff
@@ -4,18 +4,23 @@ sidebar_position: 4
 
 # Troubleshooting
 
-Here, find some answers to the question : I have downloaded Podman Desktop but I am unable to see any of my image or container
+This section of documentation provides way to troubleshoot problems with Podman Desktop.
```
Comment (slemeur) [inline website/docs/troubleshooting.md]:
```suggestion
If you cannot find your issue here or in the documentation, please fill an issue on our [repository](https://github.com/containers/podman-desktop/issues). You can also explore the [discussions](https://github.com/containers/podman-desktop/discussions) and do a search on similar issues on the [repository](https://github.com/containers/podman-desktop/issues).
```

File: website/docs/troubleshooting.md [hunk 2]
```diff
@@ -4,18 +4,23 @@ sidebar_position: 4
...
 
-## Podman
+## Podman Issues
 
-### System Requirements
+### Getting Started
```
Comment (slemeur) [inline website/docs/troubleshooting.md]:
```suggestion
### Unable to see any image or container after downloading Podman Desktop
```

Comment (slemeur) [reply website/docs/troubleshooting.md]: Are you sure that's really the case? 
Is this not something handled by pre-flight checks?

**Review by slemeur — CHANGES_REQUESTED** [review]

File: website/docs/troubleshooting.md (hunk 2 above)
Comment (deekay2310) [reply website/docs/troubleshooting.md]: I used the verbatim that existed previously on the guide. Is it subject to change?
Screen Shot 2022-11-07 at 9 50 02 AM

**Review by slemeur — APPROVED** [review]
LGTM

**Pull request merged**